src/features/Missions/Missions.css

```diff
@@ -0,0 +1,95 @@
+.container {
+display: flex;
+flex-direction: column;
+align-items: center;
+justify-content: center;
+width: 100%;
+height: 100%;
+background-color:  #f5f5f5;
+}
+
```

Comment:
Please, review indentation.

I checked the files and it seems that you don't have the linters installed, It was my mistake, I forgot to add at the development branch. But using the linters you fix this problem quick.